### PR TITLE
New page set-up/Application settings

### PR DESCRIPTION
**Pull request by sajumb**
The text for the new feature

### CODE REVIEW

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+versionFrom: 7.0.0
+---
+
+# Custom Application Settings
+
+Umbraco Cloud enables you to toggle a selection of predefined settings for each of the project's environments.
+A change of an application setting enables you to change one specific behavoir of the app service that an environment is running in. The objective with application settings are to empower you with feature or functionality that isn't available by default.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Set-Up/Application-Settings/index.md:12]:
```suggestion
A change of an application setting enables you to change one specific behavior of the app service that an environment is running in. The objective of application settings is to empower you with features or functionality that aren't available by default.
```

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Currently the only setting that currently is available is the _enable client certificate loaded fromm file system_.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Set-Up/Application-Settings/index.md:14]:
```suggestion
Currently, the only setting that is available is the _enable client certificate loaded from file system_.
```

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Enable client certificate loaded from file system Explained
+If your cloud project needs to load a client certificate (such as a X.509 certificate) at runtime you can turn on this feature for one or more environments.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Set-Up/Application-Settings/index.md:17]:
```suggestion
If your cloud project needs to load a client certificate (such as an X.509 certificate) at runtime you can turn on this feature for one or more environments.
```

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+By turning this feature on for an environment you will be able to load a client certificate as a file during run-time of your cloud project.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Set-Up/Application-Settings/index.md:18]:
```suggestion
By turning this feature on for an environment, you will be able to load a client certificate as a file during the run-time of your cloud project.
```

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+![Enable Client Certificate](Images/EnableClientCertificateLoadedFromFileSystem.gif)
+
+You can find more information on loading a certificate from a file [here](https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-certificate-in-code#load-certificate-from-file).
```
Comment (eshanrnh) [inline Umbraco-Cloud/Set-Up/Application-Settings/index.md:22]:
```suggestion
For more information on loading a certificate from a file, see the [Load certificate in Windows apps](https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-certificate-in-code#load-certificate-from-file) section in the Microsoft documentation.
```

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+:::note
+When toggling the setting you will be met by a confirmation prompt. An enabling/disabling of this feature will enforce a restart of the environment.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Set-Up/Application-Settings/index.md:25]:
```suggestion
When toggling the setting, a confirmation prompt is displayed. Enabling/disabling this feature enforces a restart of the environment.
```

**Review by eshanrnh — CHANGES_REQUESTED** [review]
The PR looks good, @sajumb 💪 

I've added the Feature availability Note to the PR along with a couple of suggestions.

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md (hunk 1 above)
Comment (sajumb) [reply Umbraco-Cloud/Set-Up/Application-Settings/index.md:12]: Great suggestions. I accept! :-)

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md (hunk 2 above)
Comment (sajumb) [reply Umbraco-Cloud/Set-Up/Application-Settings/index.md:14]: Great suggestions. I accept! :-)

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md (hunk 3 above)
Comment (sajumb) [reply Umbraco-Cloud/Set-Up/Application-Settings/index.md:17]: Is it really an X.509 and not a X.509?

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md (hunk 4 above)
Comment (sajumb) [reply Umbraco-Cloud/Set-Up/Application-Settings/index.md:18]: Great suggestions. I accept! :-)

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md (hunk 5 above)
Comment (sajumb) [reply Umbraco-Cloud/Set-Up/Application-Settings/index.md:22]: Great suggestions. I accept! :-)

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+---
+versionFrom: 7.0.0
```
[the remark this answers is not in the document]
Comment (sajumb) [reply Umbraco-Cloud/Set-Up/Application-Settings/index.md:2]: Great :-)

File: Umbraco-Cloud/Set-Up/Application-Settings/index.md (hunk 3 above)
Comment (sajumb) [reply Umbraco-Cloud/Set-Up/Application-Settings/index.md:17]: But is to, I accept :-)

**Comment by eshanrnh** [comment]
Looks good 💪 Merging it in 🙂

**Pull request merged**